ape_ganache/provider.py

```diff
@@ -138,6 +138,10 @@ def mnemonic(self) -> str:
     def number_of_accounts(self) -> int:
         return self._test_config.number_of_accounts
 
+    @property
+    def hd_path(self) -> str:
+        return self._test_config.hd_path
+
```

Comment:
If it's optional, maybe something like this?

```suggestion
    @property
    def hd_path(self) -> str:
        return getattr(self._test_config, "hd_path", "m/44'/60'/0'")

```

Reply:
It has a default value already in test_config as i updated also ape version. It will just work.

Reply:
Apologies, I now see the merged PR in core.